tests/Feature/Platform/Action/BuildGameListActionTest.php

```diff
@@ -0,0 +1,740 @@
+
+        Game::factory()->create([
+            'ID' => 1001,
+            'ConsoleID' => $systemGb->id,
+            'Title' => '~Hack~ Twitch Plays Pokemon: Anniversary Red',
+            'ImageIcon' => '/Images/094381.png',
+            'released_at' => '2015-01-01 00:00:00',
+            'released_at_granularity' => 'year',
+            'players_total' => 120,
+            'players_hardcore' => 97,
+            'achievements_published' => 47,
+            'points_total' => 300,
+            'TotalTruePoints' => 363,
+            'Updated' => Carbon::parse('2023-06-06'),
+        ]);
+
+        Game::factory()->create([
+            'ID' => 1002,
+            'ConsoleID' => $systemNes->id,
+            'Title' => 'Final Fantasy [Subset - Solo Class]',
+            'ImageIcon' => '/Images/071115.png',
+            'released_at' => '1987-12-18 00:55:31',
+            'released_at_granularity' => 'day',
+            'players_total' => 2856,
+            'players_hardcore' => 2054,
+            'achievements_published' => 144,
+            'points_total' => 1747,
+            'TotalTruePoints' => 35962,
+            'Updated' => Carbon::parse('2023-06-06'),
+        ]);
+
+        Game::factory()->create([
+            'ID' => 1003,
+            'ConsoleID' => $systemNes->id,
+            'Title' => 'Dragon Quest III | Dragon Warrior III',
+            'ImageIcon' => '/Images/026797.png',
+            'released_at' => '1992-10-15 00:00:00',
+            'released_at_granularity' => 'month',
+            'players_total' => 826,
+            'players_hardcore' => 623,
+            'achievements_published' => 50,
+            'points_total' => 400,
+            'TotalTruePoints' => 548,
+            'Updated' => Carbon::parse('2023-06-06'),
+        ]);
+
+        Game::factory()->create([
+            'ID' => 1004,
+            'ConsoleID' => $systemNes->id,
+            'Title' => 'Cycle Race: Road Man',
+            'ImageIcon' => '/Images/013746.png',
+            'released_at' => null,
+            'released_at_granularity' => null,
+            'players_total' => 0,
+            'players_hardcore' => 0,
+            'achievements_published' => 0,
+            'points_total' => 0,
+            'TotalTruePoints' => 0,
+            'Updated' => Carbon::parse('2023-06-06'),
+        ]);
+
+        Game::factory()->create([
+            'ID' => 1005,
+            'ConsoleID' => $systemNes->id,
+            'Title' => 'Double Moon Densetsu',
+            'ImageIcon' => '/Images/071237.png',
+            'released_at' => '1992-10-14 00:00:00',
+            'released_at_granularity' => 'day',
+            'players_total' => 18,
+            'players_hardcore' => 17,
+            'achievements_published' => 38,
+            'points_total' => 240,
+            'TotalTruePoints' => 282,
+            'Updated' => Carbon::parse('2023-06-06'),
+        ]);
+    }
+
+    private function addAllGamesToUserPlayList(User $user, array $gameIds = []): void
```

Comment:
This should just be `addGamesToUserPlayList`, `all` implies that all existing games would be added.

Also, why is the default an empty array? It doesn't branch into any sort of default behavior, so calling the method without providing a list of `gameIds` would do nothing, so there's no reason to call it without a list of `gameIds`.

Reply:
Renamed to `addGameIdsToUserPlayList` in latest and removed the default empty array definition.